spec/controllers/my_facilities/drug_stocks_controller_spec.rb

```diff
@@ -69,7 +69,7 @@
       let(:params) {
         {
           facility_id: facility.id,
-          for_end_of_month: Date.today.strftime("%B %Y")
+          for_end_of_month: Date.today.strftime("%b-%Y")
```

Comment:
We will extract `"%b-%Y"` into a constant in another PR since it will change a lot of files in this one.